pymbar/fes.py

```diff
@@ -399,11 +399,11 @@ def generate_fes(
                         0, N_k[k], size=N_k[k]
                     )
                     index += N_k[k]
-                    # recompute MBAR.
-                    mbar = pymbar.MBAR(
-                        self.u_kn[:, bootstrap_indices], self.N_k, initial_f_k=self.mbar.f_k
-                    )
-                    x_nb = x_n[bootstrap_indices]
+                # recompute MBAR.
```

Comment:
This was unnecessary - it was running MBAR too many times.  This saves approximately 2X time.